src/main/java/com/smartsheet/api/models/Cell.java

```diff
@@ -113,6 +113,21 @@
      */
     private Boolean overrideValidation;
 
+    /**
+     * Default constructor
+     */
+    public Cell() {
+    }
+
+    /**
+     * Construct a cell with specified column id
+     *
+     * @param id Column id
```

Comment:
This is where you were worried it could be confusing, right? Maybe make a note that the id of a Cell is its column id?

Reply:
Yes, hence the note in the javadoc